wiki/Beatmap/Pattern/osu!/csj/en.md

```diff
@@ -0,0 +1,11 @@
+---
+stub: true
+tags:
+  - CSJ
+---
+
+# Cross Screen Jumps
```

Comment:
I think adding this as a subsection to `Jump` is okay.

Reply:
I thought of that, and i also think it fits there more.
Applied